superset/assets/package.json

```diff
@@ -15,7 +15,7 @@
     "dev": "webpack --mode=development --colors --progress --debug --watch",
     "dev-server": "webpack-dev-server --mode=development --progress",
     "prod": "node --max_old_space_size=4096 webpack --mode=production --colors --progress",
```

Comment:
should we also add this to the `prod` script? or delete this script? @mistercrunch do you know why this is here? airbnb doesn't use it, and iirc it was just setting an upper limit on the process, which also makes it take like 20 min.

Reply:
`prod` is legacy at this point, we use `build`, I don't fully understand the implications / differences...

Reply:
@kristw Aha, got it. Thanks for clarifying! 👍 The change looks good to go. 🚢

Reply:
I can do whatever you both decide. Please let me know. I don't know what `prod` is for, so didn't touch it. 

Thanks @xtinec . Always great to have second eye on.

Reply:
I found an old PR from 2016 that reduced the size to what it is today.
https://github.com/apache/incubator-superset/pull/1679

Seems like there is no harm deleting it as the reasoning is weakened today with RAMs getting bigger. 😄  

We should probably update this bit in `CONTRIBUTING.md` before deleting `prod` and also make a note in `UPDATING.md` in case anyone has `npm/yarn run prod` in their build pipeline.
```
# Compile the Javascript and CSS in production/optimized mode for official releases
npm run prod
```
Thoughts? @mistercrunch @williaster @kristw 

I'm also cool with this PR shipping as-is. Going to go ahead and approve now.

Reply:
Last I checked the tradeoffs between `run build` and `run prod` were that `run prod` took much longer and was significantly smaller. I'm not sure exactly what it entails.

Reply:
Umm, how about merging this and make `dropping npm run prod` as another PR so it will be more obvious.

Reply:
sgtm